docs/ota-client-guide/modules/ROOT/pages/account-setup.adoc

```diff
@@ -8,12 +8,24 @@ We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}
 endif::[]
 
 
-In OTA Connect, all devices and software belong to one *user* login. However, you don't want to mix up test software and production software by creating them all under the same user.
+If you do not want to mix up test and production software, you may use separate user logins or create additional environments.
 
-In a proper production workflow, you'll need separate user logins to manage the different stages:
+== Separate user logins
 
-. A developer user such as "\dev@acme.com".
-. A QA user such as "\qa@acme.com".
-. A production user such as "\prod@acme.com".
+To manage the different stages of production workflow, you can create separate user logins:
 
-These logins provide you with a convenient way of clearly separating your development, QA and production resources.
+. A developer user such as "\dev@acme.com"
+. A QA user such as "\qa@acme.com"
+. A production user such as "\prod@acme.com"
+
+These logins provide you with a convenient way to separate your development, QA, and production resources.
+
+== Additional environments image:img::beta-icon.svg[Beta]
+
+NOTE: This feature is in beta and is only visible to users who are part of our beta program. If you would like to join the beta program, contact us at link:mailto:otaconnect.support@here.com[otaconnect.support@here.com] to request access.
+
+Instead of creating multiple accounts for each environment, you can now xref:create-environment.adoc[create] up to 10 additional environments using only one account. For example, you may need to have different environments for pre-production and production.
```

Comment:
I think the link for "create-environment" will produce a broken link because it is not in the aktualizr repo. You would have to reference the repository/folder it is located in. I think it will be something like ` xref:ota-plus::create-environment.adoc[create]. But I'm not sure about the repo reference. Maybe @tkfu can confirm the right one.

Reply:
It should rather be `xref:ota-web::create-environment.adoc[create]`

Reply:
Agree, my bad.
Changed